hail/hail/test/src/is/hail/expr/ir/BlockMatrixIRSuite.scala

```diff
@@ -47,6 +48,45 @@ class BlockMatrixIRSuite extends HailSuite {
     BlockMatrixMap2(left, right, l.name, r.name, ApplyBinaryPrimOp(op, l, r), strategy)
   }
 
+  @Test def testValueToBlockMatrix(): scalatest.Assertion = {
```

Comment:
Please parameterise, make property-based or split these test cases up according to data type as failures in one will hide failures in subsequent assertions.